src/Gui/Quarter/QuarterWidget.cpp

```diff
@@ -153,12 +153,7 @@ class CustomGLWidget : public QOpenGLWidget {
         Q_UNUSED(shareWidget);
         QSurfaceFormat surfaceFormat(format);
         surfaceFormat.setSwapBehavior(QSurfaceFormat::DoubleBuffer);
-        // With the settings below we could determine deprecated OpenGL API
-        // but can't do this since otherwise it will complain about almost any
-        // OpenGL call in Coin3d
-        //surfaceFormat.setMajorVersion(3);
-        //surfaceFormat.setMinorVersion(2);
-        //surfaceFormat.setProfile(QSurfaceFormat::CoreProfile);
```

Comment:
Maybe we could leave this for now, we might add it in soon.
It's also not related to qt versions